src/paulscode/android/mupen64plusae/input/map/TouchMap.java

```diff
@@ -265,6 +271,28 @@ public int getButtonPress( int xLocation, int yLocation )
     }
     
     /**
+     * Gets the frame for the N64 button with a given asset name
+     * 
+     * @param assetName The asset name for the button
+     * 
+     * @return The frame for the N64 button with the given asset name
+     * 
+     */
+    public Rect getButtonFrame( String assetName )
```

Comment:
I changed it to return drawRect, after confirming that it was identical to the Rect I was creating manually. Overlooked that in my quest to figure out why the index couldn't be mapped to buttonMasks.